spec/features/tagging_content_spec.rb

```diff
@@ -1,6 +1,15 @@
 require "rails_helper"
 
-RSpec.describe "Tagging content" do
+RSpec.describe "Tagging content", type: :feature do
+  include PublishingApiHelper
+
+  before do
```

Comment:
We've made this change, and also removed some redundant stubs we added by accident.